internal/assets/cli-templates/component_config.yaml

```diff
@@ -1,6 +1,8 @@
 # LATEST_VERSION
 components:
   cert-manager: "2.2.2_release"
-  # local-path-provisioner: "v0.0.28"
-  # oceanbase-dashboard: "0.2.1"
+  local-path-provisioner: "v0.0.28"
+  local-path-provisioner-dev: "master"
+  ob-dashboard: "0.3.0"
   ob-operator: "2.2.2_release"
+  ob-operator-dev: "master"
```

Comment:
`ob-operator-dev` and `local-path-provisioner-dev` should be removed.

Reply:
`dev` means development  version of components, maybe they should be reserved to provide more choice.